### PR TITLE
Upgrad maven to 3.2.5

### PR DESCRIPTION
**Pull request by ouaibsky**

### CODE REVIEW

File: atlassian-ide-plugin.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<atlassian-ide-plugin>
```
Comment (olamy) [inline atlassian-ide-plugin.xml]: why this file?

**Comment by olamy** [comment]
this upgrade is sadly as usual not as simple as that.
Have a look at the failing tests.

**Comment by ouaibsky** [comment]
ok I'm looking, facing some issue to download all dep from jenkins repo (my company policies for proxy is a nightmare :-(

**Comment by ouaibsky** [comment]
About Test that failed due to: 
```
Caused by: java.lang.NoSuchMethodError: org.apache.maven.cli.CLIReportingUtils.formatDuration(J)Ljava/lang/String;
```

I don't know why, but we still have a dep on maven-embedder:3.1.1 and we need 3.2.5.
the 3.2.5 is well provided in pom and when I open project into intellij, I can see the right version.

So I guess there is something wrong when test are started and classpath is incorrect.

**Comment by ouaibsky** [comment]
I'm sorry if I push commits that break the continuous build, but looks like there is some errors specific to windows (platform I'm developing on at office) so I'm not able to build locally something without errors.

File: pom.xml [hunk 2]
```diff
@@ -98,7 +98,7 @@ THE SOFTWARE.
       <artifactId>maven-agent</artifactId>
       <version>${mavenInterceptorsVersion}</version>
       <exclusions>
-        <exclusion>
+        <exclusion>plexus-utils
```
Comment (ldez) [inline pom.xml]: wrong copy/paste

**Comment by olamy** [comment]
very old and not really needed anymore. so closing it
feel free to reopen it and update it if still some need...

**Pull request closed without merging**